scripts/installPackages.js

```diff
@@ -1,14 +1,20 @@
 // install_packages.js
 // made by Homura
+// refactored by LeonardoBagi
 const chalk = require("chalk");
-const fs = require("fs");
 const childProcess = require("child_process");
 
 const package = require("../package.json");
-const missing = Object.keys(package.dependencies)
-    .filter((package) => !fs.existsSync(`../node_modules/${package}`));
+const missing = Object.keys(package.dependencies).filter(package => {
+    try {
+        require.resolve(package);
+        return false;
+    } catch {
+        return true;
+    }
+});
 
 if (missing.length > 0) {
     console.log(chalk.hex("#79b0fc")("Installing Packages: ") + missing.join(", "));
-    childProcess.exec(`npm i --save ${missing.join(" ")}`);
-}
\ No newline at end of file
+    childProcess.execSync(`npm i ${missing.join(" ")}`);
```

Comment:
These changes are definitely out of what we're trying to add with this PR. I'd suggest moving them into another PR or just removing them entirely. This also includes the changes done with `package.json`, and `generator.js`